pvm.py

```diff
@@ -43,9 +43,13 @@ def parse_commands(*args):
 		pass
 	# TODO: Create another python file to control two display
 	if command=="file":
-		_logger.info("File set: %s", PEFIX_PATH + value)
+		_logger.info("File set: %s", PREFIX_PATH + value)
+		if IS_FILE_SET:
+			_logger.info("The file has already been set!")
```

Comment:
Maybe you can add a solution to this log output. Let users know that they can click stop to unset file.

Reply:
I would also suggest:
```suggestion
			_logger.info("Stopping media, and loading video file")
```
or `replacing` instead of `loading`

Reply:
This has been changed. Resolve this now.